omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/OrderFrequencyController1_10Test.java

```diff
@@ -150,4 +155,29 @@ public void shouldSearchAndReturnAListOfOrderFrequencysMatchingTheQueryString()
 		assertThat(uuids, hasItems(expectedUuids));
 	}
 	
+	@Test
+	public void shouldCreateAnOrderFrequency() throws Exception {
+		String json = "{ \"names\": [{\"name\":\"Frequency test\", \"locale\":\"en\", \"conceptNameType\":\""
+		        + ConceptNameType.FULLY_SPECIFIED
+		        + "\"}], \"datatype\":\"8d4a4488-c2cc-11de-8d13-0010c6dffd0f\", \"conceptClass\":\"Frequency\" }";
```

Comment:
Why don't you make this test simpler by using an existing concept instead of creating one?

Reply:
The existing `Concept` related to Frequency in the current `standardTestDataset.xml` is retire. I just felt it logical to create one that matches this use-case.

Reply:
Does it matter which concept you use?

Reply:
It really matters, though this time not because of it being retired. I get ValidationException saying concept cannot be shared when I try the available alternative.

Reply:
Can you share the error details and stack trace when you use an existing concept?

Reply:
Sure @dkayiwa , here is the error and stack trace  https://pastebin.com/FrRR3gmB .

Reply:
How many concepts did you try?

Reply:
The concept referenced is the only one existing related to frequency/order frequency.

Reply:
What happens when you choose a non related one?

Reply:
I got ValidationException errors saying "concept must be of Frequency ConceptClass" .

Reply:
Are you sure that conceptClass takes the name of the class instead of the uuid?

Reply:
Yes @dkayiwa . That's why the test doesn't fail.